web/packages/teleterm/src/ui/DocumentCluster/UnifiedResources.tsx

```diff
@@ -190,14 +205,55 @@ function Resources(props: {
     });
   }
 
+  function getPinning(): UnifiedResourcesPinning {
+    // optimistically assume that pinning is supported
```

Comment:
probably a good idea as there isn't a way to "disable" pinning currently, and is only as a way to support old clusters. but since this is coming out in 15(?), the n-1 versions should all support pinning

Reply:
This comes out in v15, but pinning has been added in v14.1.5. We still have to support v14.0.0.